src/app/client/src/app/modules/shared/components/collection-player-metadata/collection-player-metadata.component.spec.ts

```diff
@@ -35,4 +37,13 @@ describe('CollectionPlayerMetadataComponent', () => {
     fixture.detectChanges();
     expect(component.collectionId).toBe('LP_FT_TextBook2');
   });
+
+  it('should open popup on click of show content credits with contentCredits data', () => {
+    component.metaData = Response.metaData;
+    spyOn(component, 'showContentCredits').and.callThrough();
+    component.showContentCredits();
+    fixture.detectChanges();
+    expect(component.contentCreditsData).toBeDefined();
+    expect(component.showContentCreditsModal).toBeTruthy();
```

Comment:
More test assertions are required as per the implementation and test cases should cover all the possibilities of the code

Reply:
done